client/src/main/java/io/pravega/client/segment/impl/SegmentInputStream.java

```diff
@@ -40,41 +39,30 @@
     public abstract long getOffset();
 
     /**
-     * Reads bytes from the segment a single event. Buffering is performed internally to try to prevent
-     * blocking. If there is no event after timeout null will be returned. EndOfSegmentException indicates the
-     * segment has ended an no more events may be read.
+     * If data is available this will copy bytes from an internal buffer into the buffer provided.
+     * If the provided buffer cannot be fully filled it will return the data it has. If no data is
```

Comment:
`fully filled it will return the data it has` -> `fully filled, it will return the data it has`